src/binding/python/Series.cpp

```diff
@@ -131,6 +131,8 @@ void init_Series(py::module &m) {
         .def("set_particles_path", &Series::setParticlesPath)
         .def_property_readonly("author", &Series::author)
         .def("set_author", &Series::setAuthor)
+        .def_property_readonly("machine", &Series::machine)
+        .def("set_machine", &Series::setMachine)
```

Comment:
Very nice, thanks a lot!
I try to transition to read-write properties, because that's a bit more Pythonic (sorry for the dated existing properties in Series).

Can we please set this as
```suggestion
        .def_property("machine",
            &Series::machine,
            &Series::setMachine,
            "Indicate the machine or relevant hardware that created the file.")
```

With that, we are able to read and write them as this:
```py
s.machine = "hemera-hzdr"
print(s.machine)
```

As a last step, please pick a read-write test in `test/python/API/APITest.py` and add two lines (one writing to a series, one reading and asserting the value in the read) to check if writing and reading this works as expected.

Thanks a lot! :tada:

Reply:
@ax3l I will apply your suggestion. I think this will require an update of the docs as well.

Reply:
Yep, did only put that in the summary: the file `docs/source/usage/firstwrite.rst` needs a slight update.